src/main/java/org/broadinstitute/hellbender/engine/filters/PlatformReadFilter.java

```diff
@@ -21,7 +21,7 @@
     private static final long serialVersionUID = 1L;
 
     @Argument(fullName = "PLFilterName", shortName = "PLFilterName",
-            doc="Keep reads with RG:PL attribute containing this string", optional=false)
+            doc="Keep reads with RG:PL attribute containing this string.", optional=false)
```

Comment:
Filter description has typo at line 15: "th"

Also, should clarify that "the PL attribute" refers to a read group attribute. And the explanation should be rephrased to a proper sentence, meaning we shouldn't have the meaningful part thrown in as a subclause in parentheses. 

And the arg doc string should explain the arg, not the filter purpose, as previously. Should be eg "PL attribute to match"